rsciio/tiff/_api.py

```diff
@@ -341,7 +350,7 @@ def _read_serie(
         )
 
     metadata_mapping = get_metadata_mapping(page, op)
-    if "SightX_Notes" in op:
+    if "SightX_Notes" in op:   # TODO: Why is this here?
```

Comment:
Most of vendor metadata mapping is taken care in `get_metadata_mapping()` function. Why this JEOL-tiff specific exception code is here instead of being at jeol metadata mapping section? that  is what I mean. I feel urge to try moving it to get_metadata_mapping, but seeing currently ongoing test failures with this sigh X, I rather don't want to touch it.

Reply:
Ok, make sense.
Some of the failure are related to a recent release of `python-box` and others are because of `imageio` 2.28 released today!